silk/datadog_checks/silk/events.py

```diff
@@ -0,0 +1,22 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+ALERT_TYPES = {"INFO": "info", "ERROR": "error", "WARNING": "warning", "CRITICAL": "error"}
+
+
+class SilkEvent(object):
+    def __init__(self, raw_event, tags):
+        self.payload = {
```

Comment:
Can we do this validation inside of the silk event class?

Reply:
I originally added it inside the event class, although logging any errors/warnings didn't make much sense in there since I would need to import the logger into each event. I felt it would make more sense to keep the responsibility of the event class to just making the event payload, and the validation should be handled by the main class when sending the event.

Reply:
I was thinking that specific logging didn't need to happen for this, just handling edge cases which would throw exceptions when creating events. We could handle this verification in the event class, then maybe log a debug line if the event payload can't be created?

Reply:
Sounds good, I can raise an exception when creating a malformed event and then handle it as a debug line outside of the event class.